lib/gems/pending/appliance_console/database_replication_standby.rb

```diff
@@ -54,7 +70,9 @@ def ask_for_repmgrd_configuration
 
     def activate
       say("Configuring Replication Standby Server...")
-      data_dir_empty? &&
+      initialize_postgresql_disk if disk
+      PostgresAdmin.prep_data_directory &&
```

Comment:
This doesn't return a value that makes sense with this `&&` chaining. Maybe this should be part of `initialize_postgresql_disk`?